lib/oli_web/controllers/cognito_controller.ex

```diff
@@ -14,6 +14,33 @@ defmodule OliWeb.CognitoController do
 
   @jwks_relative_path "/.well-known/jwks.json"
 
+  def index(
```

Comment:
Should this function have a parameter-less version similar to:

```
  def launch(conn, params) do
    redirect_with_error(conn, params, "Missing parameters")
  end
```

Reply:
Great catch - thanks!